webui/react/src/shared/.git-blame-ignore-revs

```diff
@@ -0,0 +1,3 @@
+# .git-blame-ignore-revs
+# Format code with Prettier in web
+85396ebc4f0ec95c6b4ef9625061c92fa324cdce
```

Comment:
I think this will probably change (I'm not sure) after we push hence why I suggested we leave it for later

eg
  commit 62756f5d5a20c4ec0ca4b0339687c1085b4852f6
vs b63e05d28bd51703b1eabd1c55adeea9bc4b3d59